include/regs.h

```diff
@@ -65,6 +65,15 @@ extern RegEditor* gRegEditor;
 #define R_ENABLE_ARENA_DBG                SREG(0) // Same as OoT
 #define R_ROOM_IMAGE_NODRAW_FLAGS         SREG(25)
 #define R_UPDATE_RATE                     SREG(30)
+
+#define R_VI_MODE_EDIT_STATE              SREG(48)
+#define R_VI_MODE_EDIT_WIDTH              SREG(49)
+#define R_VI_MODE_EDIT_HEIGHT             SREG(50)
+#define R_VI_MODE_EDIT_ULY_ADJ            SREG(51)
+#define R_VI_MODE_EDIT_LRY_ADJ            SREG(52)
+#define R_VI_MODE_EDIT_ULX_ADJ            SREG(53)
+#define R_VI_MODE_EDIT_LRX_ADJ            SREG(54)
+
 #define R_FB_FILTER_TYPE                  SREG(80)
```

Comment:
```suggestion
#define R_UPDATE_RATE                     SREG(30)
#define R_VI_MODE_EDIT_STATE              SREG(48)
#define R_VI_MODE_EDIT_WIDTH              SREG(49)
#define R_VI_MODE_EDIT_HEIGHT             SREG(50)
#define R_VI_MODE_EDIT_ULY_ADJ            SREG(51)
#define R_VI_MODE_EDIT_LRY_ADJ            SREG(52)
#define R_VI_MODE_EDIT_ULX_ADJ            SREG(53)
#define R_VI_MODE_EDIT_LRX_ADJ            SREG(54)
#define R_FB_FILTER_TYPE                  SREG(80)
```
Might be fine as is, but I think spacing is to break up the different letter'd sregs instead of by function